src/main/java/ezvcard/io/json/JCardSerializer.java

```diff
@@ -0,0 +1,68 @@
+package ezvcard.io.json;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.core.*;
+import com.fasterxml.jackson.databind.*;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+
+import ezvcard.VCard;
+
+public class JCardSerializer extends StdSerializer<VCard> {
+	private static final long serialVersionUID = 1L;
+
+	public static void configureObjectMapper(ObjectMapper mapper) {
+		List<JsonSerializer<?>> list = new ArrayList<JsonSerializer<?>>();
+		list.add(new JCardSerializer());
+		mapper.registerModule(new SimpleModule("jcardserializer", Version.unknownVersion(), list));
+	}
+
+	private static boolean addProdIdDefault = true;
+	private boolean addProdId = addProdIdDefault;
+
+	/**
+	 * Sets whether or not to add a "PRODID" property to each vCard, saying that
+	 * the vCard was generated by this library. For 2.1 vCards, the extended
+	 * property "X-PRODID" will be added, since "PRODID" is not supported by
+	 * that version.
+	 * 
+	 * @param add
+	 *            the value that any new JCardSerializers will be configured
+	 *            with (defaults to true)O
+	 */
+	public static void addProdIdByDefault(boolean add) {
```

Comment:
I can add a boolean parameter to the configureObjectMapper static method, so if users choose to configure their ObjectMapper that way, it will use the chosen constructor, but if they have annotated their object with @JsonSerialize(using=JCardSerializer.class), I expect Jackson would be creating the deserializer with Class.newInstance(), so as I see it our options are:

- assign a default value in the no-arg constructor
- set the property to Boolean.valueOf(System.getProperty("ezvcard.json.addProdId"))
- use a global default value

If we go with the first option, users can always subclass JCardSerializer with something that calls the boolean constructor if they want different behavior than the default.